BUG: fix warning in 05-sim-long_jumps.c

drakenclimber opened the pull request:
I just did a build on a more current linux box and spotted this warning.  Easy fix:


Commit 3c2da115b5b35 "tests: improve 05-sim-long_jumps to work better
across arch/ABIs" introduced the following warning.  Let's fix it.

	05-sim-long_jumps.c: In function ‘main’:
	05-sim-long_jumps.c:68:25: warning: implicit declaration of function ‘free’ [-Wimplicit-function-declaration]
	   68 |                         free(syscall);

Signed-off-by: Tom Hromatka <tom.hromatka@oracle.com>

pcmoore reviewed (approved):
Looks good to me, thanks @drakenclimber.
```
Acked-by: Paul Moore <paul@paul-moore.com>
```

drakenclimber commented:
Thanks, @pcmoore.  FIxed in main in commit https://github.com/seccomp/libseccomp/commit/092c686e1bc4a64d3f876349573ca66144fa3b73.  Fixed in release-2.5 in commit https://github.com/seccomp/libseccomp/commit/7ac9ab8cb49943b3e09ce2345883a8f3258b72db
